plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/editor/context/project/ProjectContextProvider.kt

```diff
@@ -261,23 +264,35 @@ class ProjectContextProvider(val project: Project, private val encoderServer: En
     }
 
     private fun isBuildOrBin(filePath: String): Boolean {
-        val regex = Regex("""[/\\](bin|build|node_modules|venv|env|\.idea)[/\\]""", RegexOption.IGNORE_CASE)
+        val regex = Regex("""[/\\](bin|build|node_modules|venv|.venv|env|\.idea)[/\\]""", RegexOption.IGNORE_CASE)
```

Comment:
`.conda`?

Reply:
full list is here https://github.com/github/gitignore

Reply:
I was thinking to add more but worrying if there are too many of them that makes each pattern match a slow operation.